argus/config.py

```diff
@@ -129,9 +129,23 @@ def _parse_environment(self, key):
             values[section][subkey] = opt_value
         values = values_factory(config_file, values)
 
-        start_commands = self._parser.getlist(key, 'start_commands')
-        stop_commands = self._parser.getlist(key, 'stop_commands')
-        return key, values, preparer, start_commands, stop_commands
+        start_commands = self._load_commands(key, 'start_commands')
+        stop_commands = self._load_commands(key, 'stop_commands')
+        list_services_commands = self._load_commands(
+            key, 'list_services_commands')
+        filter_services_regexes = self._load_commands(
+            key, 'filter_services_regexes')
+        start_service_command = self._load_commands(key, 'start_service_command')
+        stop_service_command = self._load_commands(key, 'stop_service_command')
+        return key, values, preparer, start_commands, stop_commands, \
```

Comment:
Move the namedtuple logic from the ``environments`` property here and return an instance of the ``environment`` tuple: https://github.com/ionuthulub/argus-ci/blob/feature/rdoenv/argus/config.py#L179